src/components/WebexActivity/ActivityHeader.jsx

```diff
@@ -43,15 +44,22 @@ export function formatMessageDate(timestamp) {
  * activity timestamp.
  *
  * @param {object} props Data passed to the component
+ * @param {string} props.className  Custom CSS class to apply
  * @param {string} props.personID  ID of the activity author
  * @param {string} props.timestamp  Activity timestamp
  * @returns {object} JSX of the component
  */
-export default function ActivityHeader({personID, timestamp}) {
+export default function ActivityHeader({className, personID, timestamp}) {
   const {displayName} = usePerson(personID);
 
+  const classBaseName = `${WEBEX_COMPONENTS_CLASS_PREFIX}-activity-header`;
+  const mainClasses = {
+    [classBaseName]: true,
+    [className]: !!className,
+  };
+
```

Comment:
I'd go with a simpler:
```js
const cssClasses = {
  [`${WEBEX_COMPONENTS_CLASS_PREFIX}-activity-header`]: true,
  [className]: !!className,
};
```

as I don't see anything "base" or main" about them.

Maybe we could also have a helper to cut down the boiler plate code:
```js
cssClasses = webexComponentClasses('activity-header', className, {...other-optional-classes...});
```

Let's see what @lalli-flores has to say about this.